server/config/requestHandler.js

```diff
@@ -49,4 +49,18 @@ module.exports = {
     res.send(responseJSON);
   },
 
-};
\ No newline at end of file
+  getPhotos: function(req, res) {
+    var responseJSON = {
+      photos: Photos
```

Comment:
The photos array was originally just a small test to see whether I could successfully post the image url from local camera. Sending back the photos array was merely a convenient way to test with Postman.

Spent the greater part of the day figuring out how to actually transmit and write pictures to the file system instead of uris. Photos array isn't needed now. See commits below!